Mathlib/Topology/Maps/Basic.lean

```diff
@@ -168,6 +168,12 @@ theorem dense_iff (hf : IsInducing f) {s : Set X} :
 theorem of_subsingleton [Subsingleton X] (f : X → Y) : IsInducing f :=
   ⟨Subsingleton.elim _ _⟩
 
+theorem continuous_iff_of_surjective (ind : IsInducing f) (surj : Function.Surjective f) :
```

Comment:
Can you add a pointer to `IsQuotientMap.continuous_iff`? Or is there something more general that subsumes both?

Reply:
Good point, what I'm proving here is that a surjective inducing map is actually a quotient map. I replaced the statement.

Reply:
This is strengthened to IsOpenQuotientMap in #25242